src/redux/reducers/snackbarReducer.js

```diff
@@ -50,16 +50,17 @@ export const snackbarReducer = (state = initialState.snackbar, action) => {
           horizontal: "center",
         },
       };
+    // case ERROR_SNACKBAR:
+    //   return {
+    //     isOpen: true,
+    //     type: "error",
+    //     message: action.errMessage,
+    //     position: {
+    //       vertical: "top",
+    //       horizontal: "center",
+    //     },
+    //   };
     case ERROR_SNACKBAR:
```

Comment:
Just confirming that we want `ERROR_SNACKBAR` to fall through to `REGISTER_ERROR` case?

Reply:
Abid changed it on his branch so i left it untouched to avoid conflicts when he merges

Reply:
sounds good <3